services/notifications/pkg/command/send_email.go

```diff
@@ -0,0 +1,57 @@
+package command
+
+import (
+	"github.com/cs3org/reva/v2/pkg/events"
+	"github.com/cs3org/reva/v2/pkg/events/stream"
+	"github.com/owncloud/ocis/v2/services/notifications/pkg/config"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli/v2"
+)
+
+// SendEmail triggers the sending of grouped email notifications for daily or weekly emails.
+func SendEmail(cfg *config.Config) *cli.Command {
+	return &cli.Command{
+		Name:  "send-email",
+		Usage: "Send grouped email notifications with daily or weekly interval. Specify at least one of the flags '--daily' or '--weekly'.",
+		Flags: []cli.Flag{
+			&cli.BoolFlag{
+				Name:    "daily",
+				Aliases: []string{"d"},
+				Usage:   "Sends grouped daily email notifications.",
+			},
+			&cli.BoolFlag{
+				Name:    "weekly",
+				Aliases: []string{"w"},
+				Usage:   "Sends grouped weekly email notifications.",
```

Comment:
what does weekly mean in this context? I'd expect notifications with a creation date within the "last 7 days" / last 168 hours!?

Reply:
`daily` and `weekly` are just groups. You can trigger the sending of a group using the `notifications send-email --daily` or `notifications send-email --weekly` cli command. 
You could also trigger the weekly job every 4 days or every 8 days but you have to consider the TTL of the events in the store when triggering not that often.